src/fetchUtils.js

```diff
@@ -27,18 +27,15 @@ handlers.decode = function (response) {
   const contentType = response.headers.get('content-type');
 
   if (contentType.includes('application/json')) {
-    return response.clone().json()
+    return response.json()
       .then(data => handlers.finish(response, data))
-      .catch(e =>
-        response.text()
-          .then(data => {
-            if (!data) {
-              return handlers.finish(response, null);
-            }
-            throw e;
-          }).catch(() => {
-            throw e;
-          }));
+      .catch(e => {
+        // No content, ignore response and return success
+        if (response.status === 202) {
```

Comment:
Why 202? 204 is for No Content.

Reply:
We check for 204 already [here](https://github.com/godaddy/reduxful/pull/65/files#diff-f25b708887dae13e693faac9fc0f0c2c4774ea8627c16deba0b7545d487c020aR23-R25). I believe most of the time [202 responses](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) are going not going to have content either, but sometimes they may have a URI pointer to the future resource or job hence the need to try and `.json()` parse it.

Reply:
If there's a `content-type` header but no content, that is a bug. It's not clear why we'd allow that for 202 and not for other cases. Perhaps all 2xx status codes that get an error loading JSON should have this kind of response swallowing.

Reply:
Yes, I tend to concur it's a bug in the API/service. I'm ok with just dropping this extra status check, but I would like to see what @michaelkasper chimes in.

Reply:
Proceeding with the change - fallback to be removed as a breaking change in next major version https://github.com/godaddy/reduxful/issues/66